clang/include/clang/AST/Stmt.h

```diff
@@ -102,8 +102,10 @@ class alignas(void *) Stmt {
 
     /// The statement class.
     unsigned sClass : 8;
+    /// The checked scope specifier for the statement.
+    unsigned CSS : 2;
```

Comment:
The field `CSS` is added as a sibling field of  `sClass` in the class `StmtBitfields` - i.e. it is present for **every** "Stmt". The field `sClass` is initialized in the constructor for `Stmt` at line 1200  (`in AST/Stmt.h` on `master` branch). It is also possible to initialize the field `CSS` in the same constructor because `GetCheckedScopeInfo` (which reads `CheckingKind` that stores the currently applicable value of the checked scope specification) is a `Sema` function. This will avoid looping over the statements in the body of a compound statement, which may be unavoidable if the `CSS` field is set in `ActOnCompoundStmt`.

@dtarditi It will be great to know your thoughts on this.